complaints/views.py

```diff
@@ -104,8 +132,44 @@ def handle_unblock_request(request):
                 messages.success(
                     request, "The Complaint has been Successfully Resolved"
                 )
+                status = "Resolved"
             if request.POST.get("status") == Complaint.CANCELLED:
                 messages.success(
                     request, "The Complaint has been Successfully Cancelled"
                 )
+                status = "Cancelled"
+            email_resolve(
+                category=form_obj.category,
+                request_id=form_obj.id,
+                issue="Request",
+                user_email=request.user.email,
+                request_status=status,
+                details=form_obj.reason,
+                remark_user=form_obj.remark_to_user,
+            )
             return render(request, "registration/home.html", context={"title": "home"})
+
+
+def email_on_request(request_id, category, details, issue, user_email):
+    """Function to send email to the user and staff when he registers a new complaint or request"""
+
+    from_email = settings.EMAIL_HOST_USER
+    subject = issue + " Registered"
+    message = f"Your {issue} with reference number {request_id}, category- {category} and details- {details} has been successfully registered. Soon a technician will be alloted to look into the {issue}"
+    to_email = [user_email]
+    send_mail(subject, message, from_email, to_email, fail_silently=True)
+    subject = "New " + issue + " Registered"
+    message = f"New {issue} with reference id {request_id}, category- {category} and details- {details} has been registered. Please allocate a technician to look into the {issue}"
+    to_email = ["ccit@hyderabad.bits-pilani.ac.in"]
+    send_mail(subject, message, from_email, to_email, fail_silently=True)
+
+
+def email_resolve(
+    request_id, request_status, issue, user_email, category, details, remark_user
+):
+    """Function to send email to the user when his request or compliant gets resolved"""
+    subject = issue + " " + request_status
+    message = f"Your {issue} with reference number {request_id}, category- {category} and details- {details} has been {request_status}. Remark- {remark_user}. Contact the adminstrator for further queries."
```

Comment:
Line is too long. Split into smaller lines. Use parentheses.

Reply:
ok.